tests/integration/helpers/cluster.py

```diff
@@ -487,6 +484,19 @@ def get_client_cmd(self):
             cmd += " client"
         return cmd
 
+    # Returns the list of currently running docker containers corresponding to this ClickHouseCluster.
+    def get_running_containers(self):
+        # docker-compose names containers using the following formula:
+        # container_name = project_name + '_' + instance_name + '_1'
+        # We need to have "^/" and "$" in the "--filter name" option below to filter by exact name of the container, see
+        # https://stackoverflow.com/questions/48767760/how-to-make-docker-container-ls-f-name-filter-by-exact-name
+        filter_name = f'^/{self.project_name}_.*_1$'
+        # We want the command "docker container list" to show only containers' ID and their names, separated by colon.
+        format = '{{.ID}}:{{.Names}}'
```

Comment:
Docker has changed the default format of the output of `docker container list`. In earlier versions this commands showed only IDs of the running containers, but currently by default it shows a table with the following header:

```
CONTAINER ID   IMAGE     COMMAND   CREATED       STATUS       PORTS     NAMES
```

So now we have to specify the format in the explicit way. `'{{.ID}}:{{.Names}}'` means we want to see only containers' ID and their names, separated by colon.